cli/hzn.go

```diff
@@ -735,6 +742,12 @@ Environment Variables:
 
 		// use HZN_EXCHANGE_NODE_AUTH for -n and trim the org
 		nodeIdTok = cliutils.WithDefaultEnvVar(nodeIdTok, "HZN_EXCHANGE_NODE_AUTH")
+
```

Comment:
Please add `org = cliutils.WithDefaultEnvVar(org, "HZN_ORG_ID")`  here so that the org can get the HZN_ORG_ID env.

Reply:
Added new local org variable for version check, because assigning some value to `org` from env var is producing errors - we cannot have it set for some cases (`-o and -p are mutually exclusive with <nodeorg> and <pattern> arguments`)